src/improvements/tasks/eth/000-opcm-upgrade-v200/README.md

```diff
@@ -0,0 +1,20 @@
+# 000-opcm-upgrade-v200: Mainnet OPCM v2.0.0: op, soneium, ink
+
+Status: [DRAFT]()
+
```

Comment:
TODO: @blmalone as part of this task, we should complete this issue: https://github.com/ethereum-optimism/superchain-ops/issues/548

Reply:
It's not entirely clear to me what this means.